articles/client-platforms/angularjs.md

```diff
@@ -29,124 +31,275 @@ alias:
   pkgType: 'js'
 }) %>
 
-**If you have an existing application, follow the steps below.**
-
+### Prerequisites
 ::: panel-info System Requirements
 This tutorial and seed project have been tested with the following:
 
-* NodeJS 4.3
+* NodeJS 4.4
 * Angular version 1.5.0-rc.0
 :::
-${include('./\_callback')}
```

Comment:
This should be in the callback section. It tells users what their current callback is set to.